packages/frameworks/frameworks.json

```diff
@@ -309,6 +309,33 @@
       }
     }
   },
+  {
+    "name": "Storybook",
```

Comment:
This should be moved to the last item in the array because we only match the first framework and Storybook should be lower priority than any other framework.

Reply:
Got it. I wasnt aware order mattered. Moving it last.